src/server/api/controllers/user.controller.js

```diff
@@ -33,6 +33,21 @@ const getUserById = async id => {
   }
 };
 
+const getUsersByRole = async role => {
+  try {
+    const users = await knex("users")
+      .join("user_roles", "user_roles.id", "users.role_id")
+      .select("*")
+      .where({ role: role });
```

Comment:
You need to query by the `role_id`. Currently it does not work.

```suggestion
      .where({ role_id: role });
```

Reply:
Sorry I was using the wrong request body. This works with role name (i.e. "admin")